### PR TITLE
Updates android target framework to 8.0; updates GooglePlayServices to 11.4.2; Updates Xamarin nugets to latest stable

### PR DESCRIPTION
**Pull request by Immons**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Libraries update

### :arrow_heading_down: What is the current behavior?

Current version of Android target framework is 7.0
Current version of GooglePlayServices is 42.1021.1
Current version of Xamarin.Forms is 2.5.0.91635
Current version of Xamarin Android binding packages varies from 23 to 26

### :new: What is the new behavior (if this is a feature change)?

New version of Android target framework is 8.0
New version of GooglePlayServices is 60.1142.0
New version of Xamarin.Forms is 2.5.0.121934
New version of Xamarin Android binding packages is 26.1.0.1

### :boom: Does this PR introduce a breaking change?

Possible

### :bug: Recommendations for testing

Everything regarding GooglePlayServices - e.g. location plugin
Everything regarding Xamarin Android binding packages

### :thinking: Checklist before submitting

- [x] All projects build
- [x] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [x] Nuspec files were updated (when applicable)
- [x] Rebased onto current develop

### CODE REVIEW

File: TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj [hunk 1]
```diff
@@ -86,10 +86,10 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
-      <InProject>false</InProject>
+      <Visible>false</Visible>
     </ImageAsset>
     <ImageAsset Include="Assets.xcassets\Contents.json">
-      <InProject>false</InProject>
+      <Visible>false</Visible>
```
Comment (nickrandolph) [inline TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj]: This change shouldn't be part of this PR as it affects iOS projects

File: TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs [hunk 2]
```diff
@@ -19,5 +20,10 @@ protected override IMvxApplication CreateApp()
             Mvx.LazyConstructAndRegisterSingleton<IImageHolder, ImageHolder>();
             return new CoreApp();
         }
+
```
Comment (nickrandolph) [inline TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs]: Why is this change part of this PR and why does it throw a NotImplementedException?

File: TestProjects/Playground/Playground.Forms.Wpf/Playground.Forms.Wpf.csproj [hunk 3]
```diff
@@ -54,7 +54,6 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
-    <Compile Include="Setup.cs" />
```
Comment (nickrandolph) [inline TestProjects/Playground/Playground.Forms.Wpf/Playground.Forms.Wpf.csproj]: I'm not sure why this file is being removed as it's not currently part of the WPF forms project - can you confirm that you've rebased off latest Develop

File: TestProjects/Playground/Playground.Forms.iOS/Playground.Forms.iOS.csproj [hunk 4]
```diff
@@ -114,11 +114,9 @@
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
       <Visible>false</Visible>
-      <InProject>false</InProject>
```
Comment (nickrandolph) [inline TestProjects/Playground/Playground.Forms.iOS/Playground.Forms.iOS.csproj]: Please revert all changes to iOS projects in this PR

File: TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj (hunk 1 above)
Comment (Immons) [reply TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj]: Well, you are right, however each time I open solution in Visual Studio 15.5.2 it is changed again - I could revert it but I think newest Visual Studio wants Visible tag instead InProject. I could also revert it and prepare new PR, but it's quite a bit of overkill - what do you think?

File: TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs (hunk 2 above)
Comment (Immons) [reply TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs]: I have tried to build all TestProjects and it wasn't implemented so the build failed. I just implemented it then with NotImplementedException, so build will pass, but if someone tries to run this PageRenderer project it will probably crash.

File: TestProjects/Playground/Playground.Forms.Wpf/Playground.Forms.Wpf.csproj (hunk 3 above)
Comment (Immons) [reply TestProjects/Playground/Playground.Forms.Wpf/Playground.Forms.Wpf.csproj]: I have rebased off the latest develop, and this file isn't it repository anymore, however still in csproj and so the TestProjects build failed.

File: TestProjects/Playground/Playground.Forms.iOS/Playground.Forms.iOS.csproj (hunk 4 above)
Comment (Immons) [reply TestProjects/Playground/Playground.Forms.iOS/Playground.Forms.iOS.csproj]: Waiting for answer at top comment.

File: TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs (hunk 2 above)
Comment (nickrandolph) [reply TestProjects/Forms/PageRenderer/PageRenderer.UWP/MvvmSetup.cs]: Hmmm, it's not causing a build failure (the test projects must be excluded from the CI build) so I think best to revert for the moment 
@Cheesebaron what do you think - leave this in this PR or shift it to provide an actual implementation for UWP?

File: TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj (hunk 1 above)
Comment (nickrandolph) [reply TestProjects/Eventhooks/Eventhooks.iOS/Eventhooks.iOS.csproj]: See https://github.com/MvvmCross/MvvmCross/pull/2493

File: TestProjects/Playground/Playground.iOS/Playground.iOS.csproj [hunk 5]
```diff
@@ -84,11 +84,9 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
-      <Visible>false</Visible>
```
Comment (nickrandolph) [inline TestProjects/Playground/Playground.iOS/Playground.iOS.csproj]: This should be moved out also

**Comment by martijn00** [comment]
I think we might have to close this PR for now. We will update all dependencies when going over to net standard.

**Comment by Immons** [comment]
@martijn00 not good, our projects crashes on Fused location with newest GPS services which we use for Vision API.

**Comment by Cheesebaron** [comment]
@martijn00 I don't see why we can't release a version before we freeze for NET Standard. These packages haven't been updated for a long time and it is due time we do it. People are obviously blocked by it.

**Comment by thefex** [comment]
@martijn00 certainly not a good move. 
That cost nothing and creates dependency hell. A lot of libraries use new Play Services -> app can be built anymore with Fusion Location plugin.
 
The only workaround is to rewrite location service by hand and stop using those libraries.

**Comment by martijn00** [comment]
You can pick up the 6.0 build with the update from Myget. Rather be helpful and submit PR's to get that stable so we can move on.

**Comment by Cheesebaron** [comment]
I asked @Immons to rebase on the latest release tag. Haven't heard from him since.
